### ISSUE TITLE
Task Scheduling

### ISSUE THREAD

**Issue by RaXorX** [kapitainsky/RcloneBrowser#73]
Not sure if it is already a feature or not, I am fairly new at using rclone or rclone browser.
Is there a possibility to schedule individual tasks for a consistent syncing? Like every hour or half an hour. Although this would create an inconsistency if say, I modify a file on my other device before that half hour is complete. In which case, the optimal option would be for the task to keep running forever. Is that an optimal solution or am I better off using google's app on windows? 

Edit: I think the other option could be to trigger the sync upload based on the application launch, and then sync download when exit.

**Comment by kapitainsky** [comment]
There is no scheduler in Rclone Browser. I see it more as a tool to transfer data to/from cloud. And try to focus development on making it working well. 

I am not sure it would be useful. There are many other backup/archive solutions available providing functionality beyond simple file copy e.g. versioning or incremental backup. 

Me personally if I want to use cloud as a backup destination I use Arq or syncthing. Everything depends on your requirement.

**Comment by MostHated** [comment]
I know this is closed but I wanted to throw it out there that I would really like this as well. If it had a simple scheduler to run a task once an hour, day, week, on file change, etc, there would be no need for me to have another application. I already have this setup with my NAS as a destination, as well as my cloud locations. It already has tasks. The only thing missing from making this a very well rounded backup solution is a way to schedule those tasks.

**Comment by kapitainsky** [comment]
Perfect timing. I am working on this for the last few weeks. It will be included in the next release soon. Scheduler and many other improvements

**Comment by MostHated** [comment]
This is great news! Very glad to hear. I look forward to the release!

**Comment by WillGunn** [comment]
In a similar vein, being able to schedule run all tasks (with an option to do 1 by 1 or n at a time in parallel) would be great.

**Comment by xianx** [comment]
Yes, Scheduler feature should be added as it will help user to do backup on regular basis.

**Comment by RaXorX** [comment]
Glad to see it was reopened and already in wip. :)

**Comment by soredake** [comment]
https://www.freedesktop.org/software/systemd/man/systemd.timer.html

**Comment by kapitainsky** [comment]
Thank you - but it would only apply to Linux/MacOS - there is no system on Windows
